web/models/packages.ex

```diff
@@ -6,9 +6,18 @@ defmodule HexWeb.Packages do
   end
 
   def get(name) do
+    Repo.get_by(Package, name: name)
+  end
+
+  def get!(name) do
```

Comment:
Having `Packages.get` and `Packages.get!` makes me think we should probably rename e.g. `Keys.get -> Keys.get!` etc in all other places. WDYT?